src/pages/tutorials/aiops/scopeBasedGrouping/index.mdx

```diff
@@ -50,7 +50,7 @@ You should end up with something like the following:
 
 ## 7-3: Inject some sample Service events
 
-Create scope_events.sh using below script to insert sample events.
+Create scope_events.sh using below script to insert sample events. Please ensure you are logged into your probe server (pi-template) and your current OCP project is "noi" before executing the scripts below.
```

Comment:
@barmansurajit let's remove reference to pi-template in the instructions so we can keep them generic in case a reader is setting up their own infrastructure to follow these instructions.

Reply:
Removed reference to pi-template